README.md

```diff
@@ -42,6 +42,7 @@ Developed by [Torchbox](https://torchbox.com/) and sponsored by [The Motley Fool
 
   `WAGTAILTRANSFER_SECRET_KEY` and the per-source `SECRET_KEY` settings are used to authenticate the communication between the source and destination instances; this prevents unauthorised users from using this API to retrieve sensitive data such as password hashes. The `SECRET_KEY` for each entry in `WAGTAILTRANSFER_SOURCES` must match that instance's `WAGTAILTRANSFER_SECRET_KEY`.
 
+* Add the the permission "Can import pages and snippets from other sites" to any users you want to be able to use this feature. This can be by editing a user's account directly, or for example by creating a "Content Importers" group, and making users a member of that group.
```

Comment:
```suggestion
* Add the permission "Can import pages and snippets from other sites" to any users you want to be able to use this feature. This can be by editing a user's account directly, or for example by creating a "Content Importers" group, and making users a member of that group.
```

One extra small point: while I know Django has user-specific permissions, I'm not aware of any way to add these in the Wagtail admin, where the standard approach is via groups only. I think it would probably be easier just to mention the groups route only, and just mention that you can add the import permission via the Settings -> Groups section of the Wagtail Admin.

Reply:
You're quite right, :facepalm:. I was too fixated on the unit tests (which do test permissions assigned directly to users) to remember what the Wagtail admin looks like. This is updated again in e288ea0.